maintenance_curation.pt.Rmd

```diff
@@ -6,213 +6,215 @@ aliases:
 # Política de curadoria de pacotes {#curationpolicy}
 
 ```{block, type="summaryblock"}
-Este capítulo resume uma proposta de política de curadoria para a manutenção contínua de pacotes
-manutenção contínua de pacotes desenvolvidos como parte das atividades do rOpenSci
-e/ou sob a organização do rOpenSci no GitHub. Essa política de curadoria visa
-apoiar esses objetivos:
+Este capítulo resume uma proposta de política de curadoria para a manutenção contínua de
+pacotes desenvolvidos como parte das atividades da rOpenSci
+e/ou sob a organização da rOpenSci no GitHub. Essa política de curadoria visa
+apoiar os seguintes objetivos:
 
 - Garantir que os pacotes fornecidos pela rOpenSci estejam atualizados e sejam de alta qualidade
 
 - Fornecer clareza quanto ao status de desenvolvimento e revisão
-    de qualquer software nos repositórios do rOpenSci
+    de qualquer software nos repositórios da rOpenSci
 
-- Gerenciar o esforço de manutenção para a equipe do rOpenSci, autores de pacotes e
-    colaboradores voluntários
+- Gerenciar o esforço de manutenção para a equipe da rOpenSci, para os(as) autores(as) de pacotes e
+    para os(as) colaboradores(as) voluntários(as)
 
 - Fornecer um mecanismo para que os pacotes sejam descontinuados de forma adequada, mantendo
     o selo de revisão por pares
```

Comment:
I see a alignment difference of two whitespaces between the first line and the second in the same bullet list item (**this applies to this an all other itens in this bullet list**). This doesn't happen in the [source file](https://github.com/ropensci/dev_guide/blob/a51db6b5fe7bfb378b2eed2cb9e2fbea36fb6426/maintenance_curation.Rmd).

I don't know if this have any impact at all (I don't know how to test this, sorry), so I'm bringing this up to be fix this alignment *if needed/wanted*.

```suggestion
-   Fornecer um mecanismo para que os pacotes sejam descontinuados de forma adequada, mantendo
    o selo de revisão por pares
```

Reply:
It does not create any impact in the end result.